### PR TITLE
Added basic ListRunningContainers func

### LINKED ISSUE #7: Test go-dockerclient
Get a feel for the library and how useful it is.
Try it out on ListContainers end-point. See if we can return suitable output without too much work/trade off.

### PR DESCRIPTION
**Pull request by Rucknar**
* Made use of go-dockerclient library
* Added docker socket to config

Overall, the library seems like a good fit for what's needed, at least initially.

Resolves #7

### CODE REVIEW

File: docker/container.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	"fmt"
+
+	gdc "github.com/fsouza/go-dockerclient"
+	"github.com/venting/silo/config"
+)
+
+// ListRunningContainers returns the effective output of a `docker ps` for running containers only.
+func ListRunningContainers(cfg config.Config) ([]gdc.APIContainers, error) {
+
+	cl := []gdc.APIContainers{}
```
Comment (flock3) [inline docker/container.go]: Given that client.ListContainers returns a slice of APIContainers{} can you not just directly return cs without the append of each value into cl?

File: docker/container.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+import (
+	"fmt"
+
+	gdc "github.com/fsouza/go-dockerclient"
+	"github.com/venting/silo/config"
+)
+
+// ListRunningContainers returns the effective output of a `docker ps` for running containers only.
+func ListRunningContainers(cfg config.Config) ([]gdc.APIContainers, error) {
```
Comment (flock3) [inline docker/container.go]: Given that ListRunningContainers only uses the docker socket var from cfg, do we want to just pass that?

Comment (Rucknar) [inline docker/container.go]: Good point, can't actually imagine needing more than the socket for this type of function.

**Comment by Rucknar** [comment]
@flock3 Updated following feedback

**Pull request merged**